Newsapp-demo-iOS/controller/ViewController.swift

```diff
@@ -11,36 +11,29 @@ import UIKit
 class ViewController: UIViewController {
 
     @IBOutlet weak var osTableView: UITableView!
-    
-    
     var osList = [
-        OperatingSystem(name :  "Mac Os", image : "apple", description : "Big Sur"),
-        OperatingSystem(name :  "Windows", image : "window", description : "Windwos 10"),
-        OperatingSystem(name :  "Kali Linux", image : "linux", description : "Kali 2020.2"),
-        OperatingSystem(name :  "Ubuntu", image : "ubuntu", description : "Ubuntu 18.04")
-        
+        OperatingSystem(name: "Mac Os", image: "apple", description: "Big Sur"),
+        OperatingSystem(name: "Windows", image: "window", description: "Windwos 10"),
+        OperatingSystem(name: "Kali Linux", image: "linux", description: "Kali 2020.2"),
+        OperatingSystem(name: "Ubuntu", image: "ubuntu", description: "Ubuntu 18.04")
     ]
-    
     override func viewDidLoad() {
         super.viewDidLoad()
         // Do any additional setup after loading the view.
          osTableView.dataSource = self
     }
 }
-
-
-extension ViewController : UITableViewDataSource{
-    
+extension ViewController: UITableViewDataSource {
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
         return osList.count
     }
-    
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        // swiftlint:disable force_cast
```

Comment:
I don't think you should disable lint for force cast

Reply:
without disabling lint for force casting, app doesn't compile...otherwise, we need to change lint rule from error to warning

Reply:
You can use following options to check nil:
1.optional binding
2.guard let
or any other way you find